team-contract.md

```diff
@@ -0,0 +1,45 @@
+# Team Contract (Project-Avalon)
+
+_Date: 2023-11-10_
+
+## Expectations
+
+The time expected for group members to work on DSCI 522 will be 8-10 hours per week/per person, unless all the team members agree to lengthen the time expected. If team members cannot complete their sections within the maximum hours, members should give notice on Slack at least 24 hours before the group submission date (Saturday 6am) to see if work should be redistributed or if the expected time should be increased.
+
+All submissions are expected to meet the criteria of the milestones as set out in the instructions. This includes making sure all scripts are working, creating tests for all your functions, and using proper grammar in written submissions.
+
+## Meetings
+
+Meeting times were established during the initial team meeting held on 2023-10-09. Meetings will be held on Friday.
+
+The group will have a second formal meeting during the lab for 522 on Monday at 2 pm. All members are expected to attend the meetings.
```

Comment:
I think instead of individual meetings, we need to outline when and how frequent there would be regular meetings.

Reply:
We can revise it and add date to it

Reply:
OK, I'll modify these

Reply:
Modified